Gems/Atom/Feature/Common/Code/Source/Mesh/MeshFeatureProcessor.cpp

```diff
@@ -73,6 +73,11 @@ namespace AZ
             m_forceRebuildDrawPackets = false;
         }
 
+        TransformServiceFeatureProcessorInterface::ObjectId MeshFeatureProcessor::GetObjectId(const MeshHandle& meshHandle) const
+        {
+            return meshHandle->m_objectId;
```

Comment:
is it possible that the handle will not be valid? should it have a validity check?

Reply:
Great question, @amzn-tommy would be a good person to field that one

Reply:
Fixed 👍 

```
if (meshHandle.IsValid())
{
    return meshHandle->m_objectId;
}

return TransformServiceFeatureProcessorInterface::ObjectId::Null;
```